packages/mesa_utils.rb

```diff
@@ -0,0 +1,28 @@
+require 'package'
+
+class Mesa_utils< Package
+  description 'Open-source implementation of the OpenGL specification - Provides tools such as glxinfo, glxgears'
+  homepage 'https://www.mesa3d.org'
+  version '8.4.0'
+  source_url 'ftp://ftp.freedesktop.org/pub/mesa/demos/mesa-demos-8.4.0.tar.bz2'
```

Comment:
Better source url? `source_url 'https://mesa.freedesktop.org/archive/demos/mesa-demos-8.4.0.tar.bz2'`

Reply:
FTP sources shall be avoided ? Why ? no problem to change this

Reply:
OK. I linked to the gitlab repository since the `mesa.freedesktop.org` didn't respond. This implies to add an additional step to the build.

Reply:
The source url works fine for me.  Where are you located?

Reply:
Ftp is insecure, slow and often unreliable.  We always want to try to use https, if possible, for security and reliability.